sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AnalyzeSentimentOptions.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.models;
+
+import com.azure.core.annotation.Fluent;
+
+/**
+ * The {@link AnalyzeSentimentOptions} model.
+ */
+@Fluent
+public final class AnalyzeSentimentOptions {
+    private TextAnalyticsRequestOptions requestOptions;
+
+    /*
+     * The boolean indicator to include opinion mining data in the returned result. If this
+     * flag is specified, you'll get a {@code minedOpinions} property on SentenceSentiment. It's available start from
+     * v3.1-preview.1 service version.
+     */
+    private boolean includeOpinionMining;
+
+    /**
+     * Get the value of {@code requestOptions}.
+     *
+     * @return the value of {@code requestOptions}.
+     */
+    public TextAnalyticsRequestOptions getRequestOptions() {
+        return requestOptions;
+    }
+
+    /**
+     * Set the value of {@code includeOpinionMining}.
+     *
+     * @param requestOptions It used to configure the scoring model for documents and show statistics.
+     * @return the AnalyzeSentimentOptions object itself.
+     */
+    public AnalyzeSentimentOptions setRequestOptions(TextAnalyticsRequestOptions requestOptions) {
+        this.requestOptions = requestOptions;
+        return this;
+    }
+
+    /**
+     * Get the value of {@code includeOpinionMining}.
```

Comment:
Need more description explaining the property. [This](https://github.com/Azure/azure-sdk-for-java/pull/13333/files#diff-6a5ac9837af9706a43bce53fad8d6ef9R15) descriptions here is for the private property and hence won't be accessible to th users. Consider moving that here.

Reply:
updated